### PR TITLE
Add a Team page with galleries for each theme

### LINKED ISSUE #414: Team galleries
After merging #413, consider adding team galleries.

### PR DESCRIPTION
**Pull request by tupui**
Closes #414

Does a bit more than just add the teams (also not sure how to put this elsewhere with the links/template).

- Add some information in the readme
- Change navbar to match with NumPy
- Add teams (and how to update)
- Update theme (before the auto-news commit)
- Remove NumPy news

### CODE REVIEW

File: README.md
Comment (stefanv) [inline README.md:3]:
```suggestion
The SciPy website is built on the 
```

File: README.md
Comment (stefanv) [inline README.md:27]:
```suggestion
To update the teams, run `team_query.py` provided by the theme:
```

File: README.md
Comment (stefanv) [inline README.md:30]: Where should maintainers.md go?  Might be easier to add a Makefile target for this.

Need to explain that they need a GH_TOKEN variable exported and [where to get that token from](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).  It needs `read:org` permissions.

File: content/en/teams.md
Comment (stefanv) [inline content/en/teams.md:6]: I suggest using the script to put each team into a separate file, and to then use an include shortcode to pull them in.  Since we don't have an include shortcode yet, I will add one to the theme now.

**Review by stefanv — COMMENTED** [review]
Peter Bell's entry isn't rendering correctly (full name is "None").  Also, my name has mysteriously disappeared from the maintainer's team :)

**Comment by tupui** [comment]
> Peter Bell's entry isn't rendering correctly (full name is "None"). Also, my name has mysteriously disappeared from the maintainer's team :)

Oups sorry about that 😅 I did not edit the files and assumed the output was correct. I will double check the entries then.

**Comment by tupui** [comment]
Actually I get what happened. Seems like you are in the org but not the teams 🤷‍♂️ There are lot of names I don't know in the whole org so I get I can add a few missing names manually until we update the teams.

**Comment by stefanv** [comment]
> Actually I get what happened. Seems like you are in the org but not the teams man_shrugging Is this to make the distinction between emeritus and active maintainers? There are lot of names I don't know in the whole org so I gess I can add a few missing names manually until we update the teams. We also have Tirth who's in the 2 teams.

Easiest might be to add my name back to the team; you shouldn't have to go around editing things by hand.

> For Peter Bell's name I can see he did not write a name on his profile. Maybe the script could use the GH handle instead of None in such case?

Good idea, I can implement that!

**Comment by tupui** [comment]
> Easiest might be to add my name back to the team; you shouldn't have to go around editing things by hand.

Agreed. I don't have any right as I am just a member, but I see you're owner so you should be able to do it.

**Comment by tupui** [comment]
@stefanv I updated the teams with the new script and include shortcode. But I did not get how to get the right path.

**Comment by stefanv** [comment]
@tupui Right now, the shortcode relies on a page being content.  I wasn't able to get it to work with non-page files.  The shortcode would be something like:

```
{{ $file := .Get 0 }}
{{ $file | readFile | safeHTML }}
```

And then use the markdown-parsing shortcode invocation:

```
{{% include-md "static/gallery.md" %}}
```

**Comment by tupui** [comment]
> @tupui Right now, the shortcode relies on a page being content. I didn't try to get it to work with non-page files. 

@stefanv So what should we do for now? Put everything in `content/en`?

**Comment by jarrodmillman** [comment]
@tupui Stefan and I are looking into this now.  Stefan's initial suggestion didn't work b/c the markdown isn't rendered to html.  He is going to have the script output HTML that we will store in static/gallery or static/teams or something.  I will test it on scientific-python.org and let you know when it is working.

**Comment by tupui** [comment]
> @tupui Stefan and I are looking into this now. Stefan's initial suggestion didn't work b/c the markdown isn't rendered to html. He is going to have the script output HTML that we will store in static/gallery or static/teams or something. I will test it on scientific-python.org and let you know when it is working.

Great thanks 🙇‍♂️

**Comment by jarrodmillman** [comment]
See https://github.com/scientific-python/scientific-python-hugo-theme/pull/57 and https://github.com/scientific-python/scientific-python.org/pull/129.

**Comment by tupui** [comment]
I have update the teams with what you did on the theme side, thanks! I also made a PR to fix the None case which was not ported when going from md to HTML.

https://github.com/scientific-python/scientific-python-hugo-theme/pull/62

File: static/teams/website.html
Comment (jarrodmillman) [inline static/teams/website.html:3]: This should be Website (not Triage).

Comment (tupui) [reply static/teams/website.html:3]: oops yes, thanks.

**Comment by tupui** [comment]
I left out the new news setup for now as it's bringing a lot of changes which are not related here. I will do another PR for that where we can discuss how we want the structure to look like.

This one is ready for me @rgommers

**Review by jarrodmillman — APPROVED** [review]

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @tupui! I like how it looks, just a simple round image/avatar and a name. A couple of comments:
- It loaded quite slowly, not sure that's fixable or if it was the Netlify preview infra perhaps.
- There are a lot of people listed who haven't been active in years, so we need to clean up those teams and rerun this script before merging.
- One thing I'm not sure about is whether to duplicate-list people or not on other teams. For _Triage_ clearly not, but e.g. for the _website_ team that's mostly overlapping with maintainers. I think it's useful to indeed list people who do a lot on the website (and yes that includes me), I just wanted to bring it up.

**Comment by tupui** [comment]
> Thanks @tupui! I like how it looks, just a simple round image/avatar and a name. A couple of comments:
> 
> - It loaded quite slowly, not sure that's fixable or if it was the Netlify preview infra perhaps.

Locally I did not notice anything. Might be the preview. I don't know how the infra behind Netlifly looks like. I would guess they have something different for production than previews.

> - There are a lot of people listed who haven't been active in years, so we need to clean up those teams and rerun this script before merging.

Sure, let me know when I should re-run the script.

> - One thing I'm not sure about is whether to duplicate-list people or not on other teams. For _Triage_ clearly not, but e.g. for the _website_ team that's mostly overlapping with maintainers. I think it's useful to indeed list people who do a lot on the website (and yes that includes me), I just wanted to bring it up.

Agreed for triage. For the rest, I would not be surprised to have people in 2 lists.

An alternative would be to list all the people under the same section and have some kind of team badges or text bellow each name.

File: Makefile
Comment (stefanv) [inline Makefile:33]:
```suggestion
```

File: content/en/teams.md
Comment (stefanv) [inline content/en/teams.md:10]: Website team excluded from Makefile?

**Review by stefanv — APPROVED** [review]

File: Makefile
Comment (tupui) [inline Makefile:26]: Originally excluded to not delete the website file because we don't yet have the team, but I will add it to not forget.
```suggestion
TEAMS = scipy-core-team triage website
```

**Comment by stefanv** [comment]
> * There are a lot of people listed who haven't been active in years, so we need to clean up those teams and rerun this script before merging.

@rgommers Is this something that you want to do soon, or should we manually build the teams for now?

**Comment by rgommers** [comment]
FYI: working on this now.

**Comment by rgommers** [comment]
All done with changes to Teams.

**Comment by tupui** [comment]
Great thanks Ralf. I will update the PR soon (I will try to find some time tomorrow).

**Comment by tupui** [comment]
@rgommers I updated the PR. I took the liberty to manually add Stéfan and myself to the web team. Feel free to update 😄
I think it looks good. One thing we could do is change the style for emeritus as seen on other repos (no colours). Although I am not sure as people could react differently to this 🤷‍♂️

File: config.yaml
Comment (rgommers) [inline config.yaml:75]: This file should be moved to `community.md` I'd say

**Review by rgommers — APPROVED** [review]
This looks great, thanks @tupui. I had one minor comment, that's not blocking and can be done later.

> One thing we could do is change the style for emeritus as seen on other repos (no colours).

That sounds fine to me; I'm happy either way. Let's see what others think. Can be done as a follow up as well.

File: config.yaml
Comment (tupui) [reply config.yaml:75]: I agree, I had open #426 and will open a follow up PR soon.

**Comment by tupui** [comment]
Thanks Ralf. I planned to work on updating the community (following NumPy) next.

**Pull request merged**
